compiler/noirc_frontend/src/parser/errors.rs

```diff
@@ -117,7 +117,7 @@ impl std::fmt::Display for ParserError {
             let vowel = "aeiou".contains(first.chars().next().unwrap());
             write!(
                 f,
-                "Expected a{} {} but found {}",
+                "Expected a{} {} but found {}.",
```

Comment:
```suggestion
                "Expected a{} {} but found {}",
```
Mentioned in my other comment but the lexer version is already a bit different so the period is unnecessary here. Removing it for consistency with other errors